### ISSUE TITLE
Need to distinguish better between HP:0100293 and HP:0003712

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#7590]
**HPO term**
HP:0100293 and HP:0003712

**Suggested revision of textual definition**
HP:0100293 should perhaps make clear that it is a microscopic description
HP:0003712 should make more clear it is clinical and macroscopic, perhaps removing or clarifying "Increased skeletal muscle cells" to remove any impression that it is a microscopic description

**Comment by pnrobinson** [comment]
[Muscle fiber hypertrophy HP:0100293](https://hpo.jax.org/app/browse/term/HP:0100293)
vs
[Skeletal muscle hypertrophy HP:0003712](https://hpo.jax.org/app/browse/term/HP:0003712)
Your suggestion captures the intended meaning. I will look for some sources to provide a more precise definition of both.

**Comment by pnrobinson** [comment]
Adding this comment to both terms (with obvious modifications): 
Muscle hypertrophy can be assessed at the whole-muscle level (macroscopic methods) or the muscle fiber level (microscopic methods). This term refers to hypertrophy assessed at the microscopic level.

**Comment by pnrobinson** [comment]
[Muscle fiber hypertrophy HP:0100293](https://hpo.jax.org/app/browse/term/HP:0100293)
-- I am changing the primary label to Hypertrophied muscle fibers to better capture the meaning. 
Here is the new definition
Def: The presence of an increased amount of muscle fibers with an increased diameter. This feature can be ascertained by microscopic examination of a muscle biopsy sample. Together with fiber size variation and atrophied muscle fibers, and hypertrophied muscle fibers are commonly found in myopathies.

**Comment by pnrobinson** [comment]
[Skeletal muscle hypertrophy HP:0003712](https://hpo.jax.org/app/browse/term/HP:0003712)
Abnormal increase in muscle size and mass not due to training.
-- this is the new definition for now. I think we may need to go over the entire section with a neurologist and the more appropriate term could be skeletal muscle enlargement.
